substrate/zombienet-sdk/tests/staking_next/mod.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::anyhow;
+use subxt::{dynamic, dynamic::Value, ext::scale_value::Composite, OnlineClient, PolkadotConfig};
+use subxt_signer::sr25519::dev;
+use zombienet_sdk::{NetworkConfig, NetworkConfigBuilder};
+
+mod tests;
+
+/// Sets `ValidatorCount` in staking pallet to 500.
+async fn set_validator_count(
+	ah_client: &OnlineClient<PolkadotConfig>,
+	validator_count: u32,
+) -> Result<(), anyhow::Error> {
+	let set_validator_count = dynamic::tx("Staking", "set_validator_count", vec![validator_count]);
+	let sudo = dynamic::tx("Sudo", "sudo", vec![set_validator_count.into_value()]);
+	let alice = dev::alice();
```

Comment:
This will work fine, but to be more correct you should query `Sudo::key` here, see who it is, and if it is one of the dev accounts (alice, bob etc.), sign and send it, and if not fail.

In other words, you are assuming alice is sudo, which may or may not be.